README.md

```diff
@@ -42,13 +42,14 @@ Shared [Travis CI](https://travis-ci.com/), [GitHub Actions](https://docs.github
     - [maven-update-pom-version](#maven-update-pom-version)
     - [maven-release](#maven-release)
     - [nexus-create-staging](#nexus-create-staging)
-    - [helm-package-chart](#helm-package-chart)
     - [helm-unit-test](#helm-unit-test)
+    - [helm-package-chart](#helm-package-chart)
```

Comment:
Are we following some kind of order here please? I thought we were alphabetical but it does not seem true anymore :)

Reply:
now that I seen it... I will try to do a PR once we don't have any other PR open here to avoid merge conflicts 🙏🏻